src/main.cpp

```diff
@@ -655,5 +667,6 @@ void loop(){
   statusLoop();    // update sensor status GUI
   otaLoop();
   gui.pageEnd();
-  delay(400);
+  //delay(400);
```

Comment:
@danielbernalb I think that the sensor loop wait 1 second for each read, the main loop should be works well to 400ms

Reply:
With 400ms the esp32 don´t work, even with 800ms generates a continuous reset at the beginning of the code and nothing works. I think is for the change in the serial routine to the original.